<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>南京邮电大学</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pic.qrc">:/new/prefix1/Final_version/Test1_proj/pic/njupt.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>电光微院-学生成绩核算系统</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>班级最高分</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>班级最低分</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>班级平均分</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="maxScore">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextBrowser" name="minScore">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextBrowser" name="averageScore">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>读取成绩</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showAll">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>查看考试总详情</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>班级总人数</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="stuNum">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="showA">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>“优秀”详情</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showB">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>“良好”详情</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showC">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>“中等”详情</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showD">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>“及格”详情</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showE">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>“不合格”详情</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>471</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pic.qrc">:/new/prefix1/Final_version/Test1_proj/pic/background.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_8</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>maxScore</zorder>
   <zorder>minScore</zorder>
   <zorder>averageScore</zorder>
   <zorder>pushButton</zorder>
   <zorder>showAll</zorder>
   <zorder>label_7</zorder>
   <zorder>stuNum</zorder>
   <zorder>showA</zorder>
   <zorder>showB</zorder>
   <zorder>showC</zorder>
   <zorder>showD</zorder>
   <zorder>showE</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>readScore()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>14</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showAll</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showA</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showA()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>16</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showB</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showC</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showC()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>12</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showD</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showD()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showE</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showE()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>readScore()</slot>
  <slot>showAll()</slot>
  <slot>showA()</slot>
  <slot>showB()</slot>
  <slot>showC()</slot>
  <slot>showD()</slot>
  <slot>showE()</slot>
 </slots>
</ui>
